<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>160</width>
      <height>96</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QComboBox" name="trig_choose">
       <item>
        <property name="text">
         <string>Двойной угол</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Сумма</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Разность</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Угол (в градусах)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="trig_input">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="count_trig">
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>237</width>
      <height>188</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="algebra_choose">
       <item>
        <property name="text">
         <string>Дискриминант квадратного уравнения</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Сумма арифметической прогрессии</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Сумма геометрической прогрессии</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="koeff_1">
       <property name="text">
        <string>Коэффициент a</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_1"/>
     </item>
     <item>
      <widget class="QLabel" name="koeff_2">
       <property name="text">
        <string>Коэффициент b</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_2"/>
     </item>
     <item>
      <widget class="QLabel" name="koeff_3">
       <property name="text">
        <string>Свободный коэффициент c</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_3"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="count_algebra">
         <property name="text">
          <string>Рассчитать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>160</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="start">
     <item>
      <widget class="QPushButton" name="btn1">
       <property name="text">
        <string>Алгебра</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn2">
       <property name="text">
        <string>Тригонометрия</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="back" native="true">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>470</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="images" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>491</width>
     <height>531</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>8</y>
      <width>471</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="formuli">
     <item>
      <widget class="QLabel" name="img_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="img_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="img_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="img_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="img_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="img_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="img_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="exit_wid" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>470</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="exit_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
